grpc/src/test/java/com/linecorp/armeria/it/grpc/HttpJsonTranscodingTest.java

```diff
@@ -569,6 +630,79 @@ void shouldReturnMethodNotAllowed() {
         assertThat(response.status()).isEqualTo(HttpStatus.METHOD_NOT_ALLOWED);
     }
 
+    @Test
+    void shouldAcceptResponseBodyValue() {
+        final QueryParamsBuilder query = QueryParams.builder();
+        query.add("value", "value");
+        final AggregatedHttpResponse response = webClient.get("/v1/echo/response_body/value?" +
+                query.toQueryString())
+                .aggregate().join();
+        assertThat(response.contentUtf8()).isEqualTo("\"value\"");
+    }
+
+    @Test
+    void shouldAcceptResponseBodyRepeated() throws JsonProcessingException {
+        final AggregatedHttpResponse response = webClient.get(
+                "/v1/echo/response_body/repeated?array_field=value1&array_field=value2")
+                .aggregate().join();
+        final JsonNode root = mapper.readTree(response.contentUtf8());
+        assertThat(root.isArray()).isTrue();
+        assertThat(root.get(0).asText()).isEqualTo("value1");
```

Comment:
We can use `assertThatJson(...).node(...).isEqualTo(...)`

Reply:
This library is pretty clean, thanks